src/e2e-test/features/dataplex/sink/RunTimeMacro.feature

```diff
@@ -0,0 +1,162 @@
+  asset type is selected as bigquery dataset
+    When Open Datafusion Project to configure pipeline
+    Then Select plugin: "Dataplex" from the plugins list as: "Source"
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Connect plugins: "Dataplex" and "Dataplex2" to establish connection
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceNameID" with value: "dataplex.referenceName"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexEntityID" with value: "dataplex.entity"
+    And Click on the Get Schema button
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "Dataplex2"
+    And Enter input plugin property: "dataplexReferenceNameID" with value: "dataplex.referenceName"
+    And Click on the Macro button of Property: "dataplexLocationID" and set the value to: "location"
+    And Click on the Macro button of Property: "dataplexLakeID" and set the value to: "lake"
+    And Click on the Macro button of Property: "dataplexZoneID" and set the value to: "zone"
+    And Click on the Macro button of Property: "dataplexAssetID" and set the value to: "asset"
+    And Click on the Macro button of Property: "dataplexTableNameID" and set the value to: "table"
+    And Click on the Macro button of Property: "dataplexTruncateTableID" and set the value to: "truncateTable"
+    And Click on the Macro button of Property: "dataplexUpdateTableSchemaID" and set the value to: "updateTableSchema"
+    And Click on the Macro button of Property: "dataplexRequirePartitionFilterID" and set the value to: "partitionFilter"
+    And Click on the Validate button
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Enter runtime argument value "dataplex.location" for key "location"
+    And Enter runtime argument value "dataplex.lake" for key "lake"
+    And Enter runtime argument value "dataplex.zone" for key "zone"
+    And Enter runtime argument value "dataplex.assetBQ" for key "asset"
+    And Enter runtime argument value "dataplex.newTable" for key "table"
+    And Enter runtime argument value "dataplex.TruncateTableTrue" for key "truncateTable"
+    And Enter runtime argument value "dataplex.UpdateSchemaTrue" for key "updateTableSchema"
+    And Enter runtime argument value "dataplex.PartitionFilterTrue" for key "partitionFilter"
+    And Run the Pipeline in Runtime with runtime arguments
+    And Wait till pipeline is in running state
+    And Verify the pipeline status is "Succeeded"
+
+  @BATCH-TS-DATAPLEX-MACRO-03
```

Comment:
Combine @BATCH-TS-DATAPLEX-MACRO-03 and @BATCH-TS-DATAPLEX-MACRO-04 in single scenario